services/blockchain-indexer/shared/dataService/business/tokens.js

```diff
@@ -48,7 +47,6 @@ const getTokens = async (params) => {
 		throw new InvalidParamsException('tokenID based retrieval is only possible along with address');
 	}
 
-	// TODO: Add logic to retrieve symbol and name from the SDK once endpoint is available
 	if (params.tokenID && params.address) {
 		const response = await requestConnector(
 			'token_getBalance',
```

Comment:
Could we please refactor the code to use an interfaced exposed from the connector instead? Please do the same for the `token_getBalances` endpoint invocation below on L58.

In addition, could also please fix the same in the following files too?
- services/blockchain-indexer/shared/dataService/business/auth/accountInfo.js
- services/blockchain-indexer/shared/dataService/business/dpos/constants.js
- services/blockchain-indexer/shared/dataService/business/dpos/unlock.js
- services/blockchain-indexer/shared/dataService/business/dpos/votesSent.js
- services/blockchain-indexer/shared/dataService/business/legacy/legacy.js
- services/blockchain-indexer/shared/dataService/business/validator/validatorInfo.js
- services/blockchain-indexer/shared/indexer/tokenIndex.js

Reply:
Done!